src/test/webhook.spec.ts

```diff
@@ -138,9 +168,9 @@ describe("User testing", function () {
     it("should delete my webhook", async function () {
       const response = await chai
         .request(app)
-        .delete(`/webhooks/${process.env.webhookId}`)
+        .delete(`/webhooks/${webhookId}`)
         .set("content-type", "application/json")
-        .set("Authorization", `${process.env.token}`);
+        .set("Authorization", `${token}`);
 
       expect(response.error).to.be.false;
       expect(response).to.have.status(StatusCodes.OK);
```

Comment:
How about `StatusCode.NO_CONTENT` after deleting webhook?

Reply:
@arkus7 Yeah, in user, recipe, image and webhook delete endpoints were 204 statuses like you said to do, but Rafał ordered to change it back.

Reply:
"Asked" not "ordered" ;)